# Water Supply and Treatment in Suriqui, Bolivia - Preliminary Design
## **juansemails: John Sulich, Helena Harris, Jennifer Jackson**

In [1]:
from aide_design.play import*

## **Introduction** 
Suriqui is an island community on the Bolivian side of Lake Titicaca that requires a system for water supply and treatment.  Currently, the area has no reliable method of water treatment and distribution. For this reason, potential water treatment/distribution systems are being identified. The communities often rely on expensive bottled water, so this project is important to ensure that the communities have access to cheap, safe drinking water. Included below is a map of the island of Siriqui, and its four villages: Paco Chachacoma, Cayumpaya, Supicachi, and Suriqui (composed of four smaller villages, Ayjadero, Rosani, Yawirjani, and Yumani).  Populations are centered in three areas (the dashed line areas): Siriqui, Northwest, South Central.  These three population centers can either be connected by a single system, or served by three separate individual systems for each local community.  This report outlines a treatment plan for three separate systems in: Siriqui, the Northwest, and the South Central population centers.  The report focuses on a treatment plan for the largest community, Siriqui, which makes up approximately 60% of the population.  


![Image](https://image.ibb.co/iXqo5w/Screen_Shot_2017_11_09_at_9_07_37_PM.png)
![Image](https://image.ibb.co/iv64LG/Screen_Shot_2017_11_30_at_9_39_51_PM.png)

## **Chosen Design**
After considering the information available, the chosen design is to provide an indoor treatment plant only for the population center of Suriqui.  This plant will be composed of either 4, 8, or 11 separate 1 L/s prefabricated water treatment systems.  With the different flow rates, it will require the operator to work either 24, 12, or 8 hours each day respectively as there will be a storage tank to collect the excess water.

This area also has a working distribution network [1]. Electricity access to this area is not ideal as during thunderstorms the population center experiences power blackouts [2].  Because of this, the water pumps used will be run on diesel in order to avoid plant shutdown during thunderstorms.  The island would rather use lake water than well water [2], for this reason, the inlet pipe will draw water in from the lake. The water will be drawn from the lake via a diesel pump.



## **Problem Identification:** 
Is it cheaper to construct three separate small water treatment plants in the respective communities or construct one large one that is accompanied by a distribution system for the entire island?  Additionally, what sort of design specifications are most beneficial to the community? Is it better to run a treatment plant for 24 hours a day or for 12 or 8 hours a day with a storage tank? Decisions regarding design specifications include:  (1) location of plant, (2) water source and size of inlet pipe (lake, spring, or well water),  (3) hours of plant operation, (4) type of pump, (5) source of electrical power for pump (power grid or diesel pump), (6) presence of water meters in homes 

## **Problem Solution:** 
We will identify the costs relating to the construction of separate water treatment systems accounting for the decisions specified above, as well as the feasibility of properly delivering safe drinking water throughout the island.   


## **Important Constraints:**

The important constraints are to minimize cost, maintenance, and space.  To minimize costs, we have developed a design that favors upfront costs that ideally can be covered by a fund rather than later costs to be paid by taxpayers on the island.  Additionally, we will try to minimize primarily the distance the water must be pumped, and secondarily the distance between the plants and the community.  The treatment system must be capable of handling the necessary flow rates for the island.  For the village of Siriqui, the required flow rate is 3.7 L/s.  The plant must be simple to construct and easy to operate. It may also be advantageous to limit the responsibilities of the plant operator as plant operators will be expensive. This is achieved by using AguaClara prefabricated 1 L/s plants.  


## **Trade-offs:**
There are some important tradeoffs to consider for this project. For instance, the more complicated the design is the more difficult it will be to find a competent operator. Also, the more flow required the larger the plant must be. For the single plant design, only one plant must be constructed but there will need to be many utility lines. However, for the multi plant design, several plants will need to be constructed, but there will need to be fewer connecting utility lines.  Some of the communities on the island already have distribution networks; the Siriqui community's distribution is functional and transports water to every house [5]. Another trade-off we will be considering is the trade-off between upfront and annual costs, particularly in regards to running the plant for 24 vs. 12 vs. 8 hours a day.


## **Perceived operator/user specifications:**

The specifications of the plant are that the qualified operator(s) should be literate and capable of reading a linear flow orifice meter.  In addition, the operator should be able to use mass conservation in order to calculate coagulant and chlorine dosage based on plant and chemical flow measurements.

## **Major design alternatives:**

Our group is focusing on a design consisting of several different pre-fabricated plants, while our cohorts are considering a single civil work system.  The following design alternatives are:

1. **Location of plant:** Plant must be located higher than the community.  The horizontal and vertical distances between the lake and the plant (pumping distance) should be minimized to reduce energy required for pumping.  

2. **Water source and size of inlet pipe.**  Possible water sources include:
    1. Siriqui Spring & Storage Tank (only operational during rainy season, flow is unknown, water quality data available)
    2. Siriqui Well (flow unknown, no water quality data available)
    3. Lake (water quality data available for other sides of island, none for Siriqui side).
    
3. **Hours of plant operation and associated number of plants/storage tanks:** 
    1. 24 hours/day with 4 plants and no storage tank
    2. 12 hours/day with 8 plants and storage tank
    3. 8 hours/day with 11 plants and storage tank
    
4. **Type of pump:** Pump must be able to deliver the necessary amount of water.

5. **Source of power for pump:**
    1. Electrical grid on island
    2. Diesel-powered pump
    
6. **Water meters**
    1. No water meters (households pay a flat rate for water, but upfront cost for meters and installation)
    2. Water meters (households pay for the amount of water they use, typically saves money)

## **Comparison of the major design alternatives:**


Our prefabricated design will consist of several plants while our cohorts will design one civil work plant.The pre-fabricated plan consists of 4, 8 or 11 plants that each can handle smaller flow rates whereas the single system would have to be large enough to handle the combined flow rate. This decision will have to be based on the trade off in cost of a single plant and many utility connections versus the cost of several smaller plants and much fewer utility connections. Having more smaller plants also has the advantage of still producing water in other parts of the island if one of the plants were to fail. The design decision will also be based on the trade-off of annual and upfront costs associated with running the plants for 8, 12 or 24 hours a day.

1. **Location of plant:** 
Many locations can be considered.  Two notable options considered are the high elevations on the Ayhadero and Yumani/Yawirjani peninsulas.

2. **Water source and size of inlet pipe.**  
The lake is deemed to be most feasible water source, as the flows of the spring and well are probably not sufficient to provide water to the entire community.  A combination of the sources was considered, but would require additional pumps, which would be expensive and require maintenance.
    
3. **Hours of plant operation and associated number of plants/storage tanks:** 
The three options (8, 12, and 24 hour operation) are considered and costs are analyzed below.  Having more than 4 plants would increase upfront costs while decreasing operator costs which would likely have to be paid for by the island inhabitants. 
    
4. **Type of pump:** 
Many pump options can be considered. 

5. **Source of power for pump:**
A diesel-powered pump was deemed to be the only option, as electricity is very unreliable on the island.  In addition, diesel is likely already being delivered to the island to run the island's power plant.
    
6. **Water meters**
Water meters represent an upfront cost, but would likely conserve water and costs.



## **Methods**

After consultation with the other group, this group will be working on multiple 1L/s plants as the other group works on a single plant to handle the entire island.  From further inspection, the island is broken up into three distinct population centers rather than 4 arbitrary villages.  To adjust the table from from the prompt, the population centers are broken down as follows:

<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
.tg .tg-9hbo{font-weight:bold;vertical-align:top}
.tg .tg-yw4l{vertical-align:top}
</style>
<table class="tg">
  <tr>
    <th class="tg-9hbo">Parameter</th>
    <th class="tg-9hbo">Suriqui</th>
    <th class="tg-9hbo">North West</th>
    <th class="tg-9hbo">South Central</th>
  </tr>
  <tr>
    <td class="tg-9hbo">People</td>
    <td class="tg-yw4l">1810</td>
    <td class="tg-yw4l">900</td>
    <td class="tg-yw4l">180</td>
  </tr>
  <tr>
    <td class="tg-9hbo">Flow Rate (L/s)</td>
    <td class="tg-yw4l">3.7 </td>
    <td class="tg-yw4l">1.92</td>
    <td class="tg-yw4l">0.38</td>
  </tr>
</table>

Our breakdown of types of plants per population center is as follows:

<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
.tg .tg-9hbo{font-weight:bold;vertical-align:top}
.tg .tg-yw4l{vertical-align:top}
</style>
<table class="tg">

  <tr>
    <th class="tg-9hbo">Parameter</th>
    <th class="tg-9hbo">Suriqui</th>
    <th class="tg-9hbo">North West</th>
    <th class="tg-9hbo">South Central</th>
  </tr>
  <tr>
    <td class="tg-9hbo">24hr operation</td>
    <td class="tg-yw4l">4 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">2 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">1 X 1L/s Prefabricated</td>
  </tr>
  <tr>
    <td class="tg-9hbo">12hr operation</td>
    <td class="tg-yw4l">8 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">4 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">1 X 1L/s Prefabricated</td>
  </tr>
  <tr>
    <td class="tg-9hbo">8hr operation</td>
    <td class="tg-yw4l">11 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">6 X 1L/s Prefabricated</td>
    <td class="tg-yw4l">1 X 1L/s Prefabricated</td>
  </tr>
  
</table>

In [11]:
# Calculations for number of plants:

S_req_flow = 3.7*(u.L/u.s)

S_op_24hr = 24*u.hour
S_op_12hr = 12*u.hour
S_op_8hr = 8*u.hour

S_req_vol = (S_req_flow*S_op_24hr).to(u.L)
S_n_plants_12hr = (S_req_vol/S_op_12hr).to(u.L/u.s)
S_n_plants_8hr = (S_req_vol/S_op_8hr).to(u.L/u.s)

print('Siriqui: The required flow for 24 hours of operation is ',S_req_flow, 'or 4 plants.')
print('Siriqui: The required flow for 12 hours of operation is ',S_n_plants_12hr, 'or 8 plants.')
print('Siriqui: The required flow for 8 hours of operation is ',S_n_plants_8hr, 'or 11 plants.')


NW_req_flow = 1.92*(u.L/u.s)
NW_req_vol = (NW_req_flow*24*u.hour).to(u.L)
NW_n_plants_12hr = (NW_req_vol/(12*u.hour)).to(u.L/u.s)
NW_n_plants_8hr = (NW_req_vol/(8*u.hour)).to(u.L/u.s)
print('Northwest: The required flows for 24, 12, and 8 hours of operation are ', NW_req_flow, ",", NW_n_plants_12hr, ', and ', NW_n_plants_8hr, "or: 2, 4, and 6 plants, respectively.")
SC_req_flow = .38*(u.L/u.s)
SC_req_vol = (SC_req_flow*24*u.hour).to(u.L)
SC_n_plants_12hr = (SC_req_vol/(12*u.hour)).to(u.L/u.s)
SC_n_plants_8hr = (SC_req_vol/(8*u.hour)).to(u.L/u.s)
print('South Central: The required flows for 24, 12, and 8 hours of operation are ', SC_req_flow, ",", SC_n_plants_12hr, ', and ', SC_n_plants_8hr, "or: 1 plant in all cases.")

Siriqui: The required flow for 24 hours of operation is  3.7 liter / second or 4 plants.
Siriqui: The required flow for 12 hours of operation is  7.4 liter / second or 8 plants.
Siriqui: The required flow for 8 hours of operation is  11.1 liter / second or 11 plants.
Northwest: The required flows for 24, 12, and 8 hours of operation are  1.92 liter / second , 3.84 liter / second , and  5.76 liter / second or: 2, 4, and 6 plants, respectively.
South Central: The required flows for 24, 12, and 8 hours of operation are  0.38 liter / second , 0.76 liter / second , and  1.14 liter / second or: 1 plant in all cases.


### Part 1: Location of Plant

Plant must be located 5-10 meters (17-33 ft) higher than the community, whose maximum elevation is 3862 m (12,670 ft).  The horizontal and vertical distances between the lake and the plant (pumping distance) should be minimized to reduce energy required for pumping. A suitable area where the plant could be located is provided below:
![Image](https://image.ibb.co/gixQub/Screen_Shot_2017_11_30_at_9_53_43_PM.png)
![Image](https://image.ibb.co/mVBmcw/Screen_Shot_2017_12_09_at_12_59_25_PM.png)




### Part 2: Water Source

Water Quality was assessed in 3 locations: (1) Supicachi Well and (2) Lake (both on Northwest side of island), as well as (3) Lake on South Central side of island (Cuyampaya).  The following are data that exceeded Maximum Contaminant Levels (MCLs)[6]:
* (1) Supicachi Well: Total Coliforms, Iron
* (2) Lake, Northwest: Boron, Total Hardness, Sulfates, Chloride, Dissolved Solids, Sodium, Calcium, Magnesium
* (3) Lake, Central South: Boron, Total Hardness, Sulfates, Chloride, Dissolved Solids, Sodium, Calcium, Magnesium, Total Coliforms
* (4) Suriqui Spring Storage Tank: Total Coliforms
* Typical influent temperature is probably around 10 degrees C (50 deg F) 

Water quality was not assessed in the Siriqui Well nor the lake near Siriqui.

The lake is deemed to be most feasible water source, as the flows of the spring and well are unknown but likely not sufficient to provide water to the entire community. 

Water could be taken from either sides of the mountain on which the plant is located.  A rough estimate would be ~300 m (1,000 ft) of 10 cm (4 inch) PVC pipe needed to tranport water from the lake to the plant. 4 inch PVC pipe is estimated to cost [$32 per 10 feet](http://pvcpipesupplies.com/4-x-10-schedule-40-pvc-pipe-h0400400pw1000.html).

The minimum wage in Bolivia is [2,000 BOB per month](https://tradingeconomics.com/bolivia/minimum-wages)
, with a 48 hour work week.


    

In [3]:
# Part 2 Calculations

#Minimum wage is 2000BOB/month with 48hours/week
# equivalent to working 6 8-hour days, with 4 week months
min_wage_BOB_monthly = 2000
BOB_to_USD = 0.15*(u.USD)
min_wage_USD_hourly =(BOB_to_USD*(min_wage_BOB_monthly/(6*4*8)))*(1/u.hour)


# Cost of PVC from water to plants (estimated as a distance of 1,000 ft)
cost_unitpvc = (32*u.USD)/(10*u.ft)
length_pvc = 1000*u.ft
cost_pvc = cost_unitpvc*length_pvc
people_install_pvc = 3
hours_install_pvc = 8*u.hr
cost_install_pvc = people_install_pvc*hours_install_pvc*min_wage_USD_hourly
total_cost_pvc = cost_pvc+cost_install_pvc 

print('The total cost of 300 m of PVC pipe + installation is ', total_cost_pvc)
#________________________________________________________

total_upfront_24hr_pt2 = total_cost_pvc
total_upfront_12hr_pt2 = total_cost_pvc
total_upfront_8hr_pt2 = total_cost_pvc




The total cost of 300 m of PVC pipe + installation is  3238 dollar


**Summary of PVC costs**

| Hours of Operation per day | Upfront Cost for PVC     |
|:--------------------------:|:------------------------:|
|             24             |          3,238           |
|             12             |          3,238           |
|              8             |          3,238           |




### Part 3: Hours of Plant Operation & Resulting Number of Plants + Storage Tank

Hours of plant operation is an important consideration.  We decided it is better to have more plants running for less hours of the day, to maximize upfront costs instead of operator costs.  

The total upfront costs for the plants are:

(1) Cost to construct the plant in Honduras (materials + labor):
    The minimum wage in Honduras is [22.44 lempiras per hour](https://www.minimum-wage.org/international/honduras) and the cost of materials for 1 plant is $2625.91 [3].
    
(2) Cost to install the plant in Bolivia (labor)

(3) Cost of storage tank (size depends on hours of operation):
    1 10,000 Gallon tank (38,000 L) costs [$5,7000](http://www.tank-depot.com/productdetails.aspx?part=N-43132&gclid=CjwKCAiA07PRBRBJEiwAS20SIINWrL9c-LK240sf2qekm1QLi6bsDOp3gqP38kmpAFIL073Z6MbZDxoCCrEQAvD_BwE)

(4) Cost of concrete buildings to house plants: concrete costs [$9-12 per square foot](https://concrete.promatcher.com/cost/)
    
The total annual costs for the plants include the cost to employ an operator.


See Calculations below. 


In [4]:
# Part 3

# Total costs for plants = Cost to construct plants in Honduras 
# + Cost to install plants in Bolivia 
# + Cost to build properly sized storage tank
# + Cost to run plants in Bolivia

# Cost to CONSTRUCT plants: cost estimates provided by Juan Guzman
# Each plant requires 4 people ~4.5 weeks to construct
# Cost of materials for 1 plant is $2625.91
cost_materials_singleplant = 2625.91*u.USD
cost_materials_eightplants = cost_materials_singleplant*8
cost_materials_elevenplants = cost_materials_singleplant*11
cost_materials_fourplants = cost_materials_singleplant*4
# hours to construct = 4 people * 4.5 weeks * 40 hrs per week IN HONDURAS 
min_wage_Lempira_hourly = 22.44*(u.USD/u.hour)
Lempira_to_USD = 0.042
hours_construction_singleplant = 4*4.5*40*u.hour          
cost_construct_singleplant = hours_construction_singleplant*min_wage_Lempira_hourly*Lempira_to_USD
cost_construct_fourplants = cost_construct_singleplant*4
cost_construct_eightplants = cost_construct_singleplant*8
cost_construct_elevenplants = cost_construct_singleplant*11
#-----------------------------------------------------------------------------------------------------
# Cost to INSTALL plants: cost estimates provided by Monroe 
# Each plant requires 4 people ~2 days to install = 4 people * 2 days * 8 hrs/day
hours_install_singleplant = 4*2*8*u.hour    
cost_install_singleplant = hours_install_singleplant*min_wage_USD_hourly
cost_install_fourplants = cost_install_singleplant*4
cost_install_eightplants = cost_install_singleplant*8
cost_install_elevenplants = cost_install_singleplant*11

#Cost of building to house the plants (1 week to construct)
squmcost = (129.6*u.USD)/(u.m**2) 
singleplantsqum = 6*u.m**2
spaceforoperator = 3*u.m**2
floorspaceplant = singleplantsqum+spaceforoperator
building_labor = 2*1*5*8*u.hour*min_wage_USD_hourly     #labor costs = 2 people*1week*5days/week*8hrs/day
building8cost = 11*floorspaceplant*squmcost + building_labor
building12cost = 8*floorspaceplant*squmcost + building_labor
building24cost = 4*floorspaceplant*squmcost + building_labor

#-------------------------------------------------------------------------------------------
# Necessary Size of Water Storage tank 
constantflow = 3.7*u.L/u.s
flow8 = 3*constantflow #if only pumping 8 hours/day
maxflowdif8 = flow8-constantflow
timeofmaxflow8 = 8*u.hr
maxvoltankendofpump8 = (maxflowdif8*timeofmaxflow8).to(u.gal)
potentialbuffer8 = .2*maxvoltankendofpump8
tankvol8 = potentialbuffer8+maxvoltankendofpump8
print('If the pump is only active 8 hours/day, the water storage tank should be a volume of at least ',ut.sig(tankvol8,5))

flow12 = 2*constantflow #if only pumping 12 hours/day
maxflowdif12 = flow12-constantflow
timeofmaxflow12 = 12*u.hr
maxvoltankendofpump12 = (maxflowdif12*timeofmaxflow12).to(u.gal)
potentialbuffer12 = .2*maxvoltankendofpump12
tankvol12 = potentialbuffer12+maxvoltankendofpump12
print('If the pump is only active 12 hours/day, the water storage tank should be a volume of at least ',ut.sig(tankvol12,5))

#Cost of Storage Tanks
price10ktank = 5700*u.USD
buytankvol = 10000*u.gal
numtanksneeded8 = tankvol8/buytankvol
numtanksneeded8 = np.ceil(numtanksneeded8)
priceofalltanks8 = numtanksneeded8*price10ktank
print('The price of all the storage tanks for the 8 hour plant is ',ut.sig(priceofalltanks8,5))
numtanksneeded12 = tankvol12/buytankvol
numtanksneeded12 = np.ceil(numtanksneeded12)
priceofalltanks12 = numtanksneeded12*price10ktank
print('The price of all the storage tanks for the 12 hour plant is ',ut.sig(priceofalltanks12,5))


# Calculate total cost of constructing + installing plants + house for plants + storage tank
total_cost_build_eightplants = cost_materials_eightplants + cost_construct_eightplants + cost_install_eightplants + priceofalltanks12 + building12cost
total_cost_build_elevenplants = cost_materials_elevenplants + cost_construct_elevenplants + cost_install_elevenplants + priceofalltanks8 + building8cost
total_cost_build_fourplants = cost_materials_fourplants + cost_construct_fourplants + cost_install_fourplants + building24cost

#Cost to operate plants
USDsalary_8hr = min_wage_USD_hourly*8*u.hr*365
USDsalary_12hr = min_wage_USD_hourly*12*u.hr*365
USDsalary_24hr = min_wage_USD_hourly*24*u.hr*365


print('The cost associated with 24 hour operation for a year in Bolivia is ', ut.sig(USDsalary_24hr,5), 'for operation, and ',ut.sig(total_cost_build_fourplants,5), 'to construct the 4 plants')
print('The cost associated with 12 hour operation for a year in Bolivia is ', USDsalary_12hr, 'for operation, and ',ut.sig(total_cost_build_eightplants,5), 'to construct the 8 plants and their storage tank')
print('The cost associated with 8 hour operation for a year in Bolivia is ', USDsalary_8hr, 'for operation, and ',ut.sig(total_cost_build_elevenplants,5), 'to construct the 11 plants and their storage tank')
#____________________________________________
total_annual_24hr_pt3 = USDsalary_24hr
total_upfront_24hr_pt3 = total_cost_build_fourplants

total_annual_12hr_pt3 = USDsalary_12hr
total_upfront_12hr_pt3 = total_cost_build_eightplants

total_annual_8hr_pt3 = USDsalary_8hr
total_upfront_8hr_pt3 = total_cost_build_elevenplants




If the pump is only active 8 hours/day, the water storage tank should be a volume of at least  67560 gal
If the pump is only active 12 hours/day, the water storage tank should be a volume of at least  50670 gal
The price of all the storage tanks for the 8 hour plant is  39900 USD
The price of all the storage tanks for the 12 hour plant is  34200 USD
The cost associated with 24 hour operation for a year in Bolivia is  13687 USD for operation, and  18409 USD to construct the 4 plants
The cost associated with 12 hour operation for a year in Bolivia is  6844 dollar for operation, and  70892 USD to construct the 8 plants and their storage tank
The cost associated with 8 hour operation for a year in Bolivia is  4562 dollar for operation, and  90305 USD to construct the 11 plants and their storage tank


**Summary of Plant and Storage Tank Costs**


| Hours of Operation per day | Upfront Cost for plant   | Cost to Employ Operator    |
|:--------------------------:|:------------------------:|:--------------------------:|
|             24             |          18,400          |          13,700            |
|             12             |          70,900          |           6,900            |
|              8             |          90,300          |           4,600            |





### Part 4: Pump

We considered the following types of pumps [3]:

**Above Ground Pump (Primed)**
    
Pros:
    
   * Easy to maintain
   * Longer lifespan than submersed pump (~20 years)
    
Cons:
    
   * Has to be primed
    
**Submerged Pump**
    
Pros:
    
   * Doesn’t need to be primed
   * Lower capital cost
    
Cons:
    
   * Difficult to remove
   * Shorter lifespan than primed pump (~5 years)
    
**Pump in Pit Below Water Level**
    
Pros:
    
   * Can use submersible or non continuously submersible pump
    
Cons:
    
   * Have to dig a pit
   * If a non-submersible pump is used another pump is needed to pump water out of pit in case of rain 

We decided to choose a primed above ground pump for its long lifespan and easy maintenance.

A typical primed above ground pump can be found [here.](http://www.firepenny.com/CET_10hp_Hartz_Diesel_Fire_Pump_p/pfp-10hp-dsl.htm)  This pump costs $6,950.

Pump maintenance includes replacement of the diesel motor every ~5 years.  A diesel motor costs around [$400.](https://www.kohler-engine-parts.opeengines.com/-p-52259?gclid=CjwKCAiA07PRBRBJEiwAS20SIDhASoCaha0vulNSSrpNBwKwcogAVqRcKT59my-5o04PPBNSiBKJixoC08EQAvD_BwE)

The pump will be at about 3830 m (12566 ft) above sea level (the elevation of Lake Titicaca), non submerged.  It will then have to pump up approximately 18.3 m (60 ft) to the plant, which is well within the capabilities of the pump.




In [5]:
# Part 4 Calculations
flow = 11*(u.L/u.s) #The highest flow possible if the pump was only running for 8 hours/day
maxheighttopump = 60*u.ft #elevation difference between pump and plant

cost_pump = 6950*u.USD

from aide_design import pipedatabase as pipe
temp = 10*u.degC
nu=pc.viscosity_kinematic(temp)
lengthpvc = 1000*u.ft
minorlosscoefsums = 2 #high estimate
pvcroughness = 0.0015*u.mm
nominaldiam = 4*u.inch
SDR=26
diam = pipe.ID_SDR(nominaldiam,SDR)
#headloss(FlowRate, Diam, Length, Nu, PipeRough, KMinor)
headloss = pc.headloss(flow,diam,lengthpvc, nu,pvcroughness,minorlosscoefsums)

totalhead = headloss + maxheighttopump

print('The design flow of ',flow, ' must be capable of being pumped at a head of ',totalhead,)
print('The pump in question is capable of providing a flow of 13L/s at a head of 24m.')
print('The pump is therefore capable of handling the design flow and head needed.')


# Pump maintenance
cost_diesel_motor = 400*u.USD
annual_cost_motor = cost_diesel_motor/5

print('Costs for the pump include ', cost_pump, 'upfront cost and ', annual_cost_motor, 'average annual cost for maintenance.')

#--------------------------------------------------------------------
total_upfront_24hr_pt4 = cost_pump
total_annual_24hr_pt4 = annual_cost_motor

total_upfront_12hr_pt4 = cost_pump
total_annual_12hr_pt4 = annual_cost_motor

total_upfront_8hr_pt4 = cost_pump
total_annual_8hr_pt4 = annual_cost_motor


The design flow of  11 liter / second  must be capable of being pumped at a head of  22.62 meter
The pump in question is capable of providing a flow of 13L/s at a head of 24m.
The pump is therefore capable of handling the design flow and head needed.
Costs for the pump include  6950 dollar upfront cost and  80 dollar average annual cost for maintenance.


**Summary of Pump Costs**


| Hours of Operation per day | Upfront Cost for pump    | Avg Annual Cost for pump maintenance    |
|:--------------------------:|:------------------------:|:---------------------------------------:|
|             24             |          6,950           |          80                             |
|             12             |          6,950           |           80                            |
|              8             |          6,950           |           80                            |
 


### Part 5: Source of Power

Engineers in Action employees in addition to David Buck from CH2M Hill reported that electricity on the island was not consistent, and often failed during rain events [2].  Our decision is to use a diesel powered pump.  Diesel is likely already being delivered to the island to power its power plant. 

See fuel calculations below.

In [6]:
# Part 5 Calculations

#Fuel
dieselcost = 0.53*u.USD/u.gal
fuelconsrate = 0.56*u.gal/u.hr
costoffuelyr_24hr = (dieselcost*fuelconsrate*365*u.day).to(u.USD) #24 hour days
costoffuelyr_12hr = (dieselcost*fuelconsrate*365*u.day*.5).to(u.USD) #12 hour days
costoffuelyr_8hr = (dieselcost*fuelconsrate*365*u.day*(1/3)).to(u.USD) #8 hour days
print('For one year: It will cost ',costoffuelyr_8hr,'to run the pump 8 hours a day, ',costoffuelyr_12hr,'to run the pump 12 hours a day, and ',costoffuelyr_24hr,'to run the pump 24 hours a day.')

total_annual_24hr_pt5 = costoffuelyr_24hr
total_annual_12hr_pt5 = costoffuelyr_12hr
total_annual_8hr_pt5 = costoffuelyr_8hr



For one year: It will cost  866.7 dollar to run the pump 8 hours a day,  1300 dollar to run the pump 12 hours a day, and  2600 dollar to run the pump 24 hours a day.


**Summary of Fuel Costs**


| Hours of Operation per day | Annual Cost for Fuel     |
|:--------------------------:|:------------------------:|
|             24             |          2,600           |
|             12             |          1,300           |
|              8             |            870           |
 

### Part 6: Water Meters

The costs of water meters was determined by estimating the number of houses in the Siriqui community and the cost of a typical water meter.  The number of houses in the community was estimated by counting the number of houses in an area that looked like it contained approximately 1/4 of the houses (80 houses) and multiplying that by 4 (320 houses total in Siruiqui community). 

Water meters typically cost around [$25](https://www.smartcitiesdive.com/news/high-cost-of-smart-water-meters-keeps-adoption-rates-low/443905/).
For installation costs, we are estimating a [3 hour installation per house](https://www.southwestwater.co.uk/frequently-asked-questions/meters/how-long-will-it-take-to-install-a-meter/), since new units will need to be installed in each house.

In [7]:
# Part 6 Calculations
# Number of meters was determined by counting the number of homes in 1/4 of the community area and multiplying by 4
num_houses = 80*4
cost_meter = 25*u.USD
cost_320_meters = num_houses*cost_meter

# Installation cost
time_install_meter = 3*u.hour
cost_install_320_meters = time_install_meter*num_houses*min_wage_USD_hourly

total_cost_meters = cost_320_meters + cost_install_320_meters

print('The total cost of the meters + installation would be ', total_cost_meters)
#_____________________________________________________________________________________________
total_upfront_24hr_pt6 = total_cost_meters
total_upfront_12hr_pt6 = total_cost_meters
total_upfront_8hr_pt6 = total_cost_meters

The total cost of the meters + installation would be  9500 dollar


$9,500 represents a large upfront cost, but would save water and cut down on later costs for the villagers.

**Summary of Meter Costs**


| Hours of Operation per day | Upfront Cost for meters  |
|:--------------------------:|:------------------------:|
|             24             |          9,500           |
|             12             |          9,500           |
|              8             |          9,500           |

 

### Part 7: Additional costs, Chlorine and Coagulant

We decided to use commercially available liquid bleach. Cost estimates were provided by Professor Monroe. We also used the coagulant dose provided in the Mechanical and Flocculation Model Design Challenge.

In [8]:
FlowPlant = 4*u.L/u.s
StockCl2 = 51.4*(u.gram/u.L)
DoseCl2 = 2*(u.mg/u.L)
cost_bleach = 2*(u.USD/u.kg)
cost_coagulant = 2*(u.USD/u.kg)
cost_bleach_mL = cost_bleach*StockCl2

def TotalChemFlow(FlowPlant, MaxChemDose, MaxAllowStock):
    totchemflow = ((MaxChemDose*FlowPlant)/MaxAllowStock).to(u.L/u.s)
    
    return totchemflow

Total_bleach = TotalChemFlow(FlowPlant,DoseCl2,StockCl2).to(u.L/u.year)
cost_bleach_annual = (Total_bleach*cost_bleach_mL).to(u.USD/u.year)
print('The annual cost of bleach would be ',cost_bleach_annual)

Dose_coagulant = 1*(u.mg/u.L)
total_coagulant = (Dose_coagulant*FlowPlant)
cost_coagulant_annual = (total_coagulant*cost_coagulant).to(u.USD/u.year)
print('The annual cost of coagulant would be ',cost_coagulant_annual)
print('totaling ', cost_bleach_annual + cost_coagulant_annual)

#_____________________________________________
total_annual_24hr_pt7 = (cost_bleach_annual+cost_coagulant_annual)*u.year
total_annual_12hr_pt7 = (cost_bleach_annual+cost_coagulant_annual)*u.year
total_annual_8hr_pt7 = (cost_bleach_annual+cost_coagulant_annual)*u.year

The annual cost of bleach would be  504.9 dollar / year
The annual cost of coagulant would be  252.5 dollar / year
totaling  757.4 dollar / year


### Summary of Chemical Costs


| Hours of Operation per day | Annual Cost for chemicals  |
|:--------------------------:|:--------------------------:|
|             24             |          760               |
|             12             |          760               |
|              8             |          760               |



## Results: Total costs 

**Upfront costs include:** 
* Cost to build 1 L/s plants (4, 8, or 11 plants) in Honduras
* Cost to install plants in Bolivia 
* Cost of building to house 1 L/s plants 
* Cost of storage tank 
* Cost of meters + Cost to install meters
* Cost of pump
* Cost of PVC from water to plant 
* Cost to install PVC pipe

**Annual costs include:**
* Cost of diesel for pump
* Cost of chlorine 
* Cost of coagulant 
* Cost of employing plant operator
* Replacement/maintenance costs for pumping system


| Hours of Operation per day | Upfront Cost             | Annual Cost                |Total     |
|:--------------------------:|:------------------------:|:--------------------------:|:--------:|
|             24             |          38,000          |           17,100           | 55,200   |
|             12             |          90,600          |           9,000            | 99,600   |
|              8             |          110,000         |           6,300            | 116,300  |


In [9]:
# Total Upfront Cost 
#Includes pump, pipes, hose, installation, and labor


total_upfront_cost_24hr = total_upfront_24hr_pt2 + total_upfront_24hr_pt3 + total_upfront_24hr_pt4 + total_upfront_24hr_pt6
total_annual_cost_24hr = total_annual_24hr_pt3 + total_annual_24hr_pt4 + total_annual_24hr_pt5 + total_annual_24hr_pt7

total_upfront_cost_12hr = total_upfront_12hr_pt2 + total_upfront_12hr_pt3 + total_upfront_12hr_pt4 + total_upfront_12hr_pt6
total_annual_cost_12hr = total_annual_12hr_pt3 + total_annual_12hr_pt4 + total_annual_12hr_pt5 + total_annual_12hr_pt7

total_upfront_cost_8hr = total_upfront_8hr_pt2 +  total_upfront_8hr_pt3 + total_upfront_8hr_pt4 +total_upfront_8hr_pt6
total_annual_cost_8hr =  total_annual_8hr_pt3 + total_annual_8hr_pt4 + total_annual_8hr_pt5 + total_annual_8hr_pt7

print('Upfront, Annual, and Total costs for 24 hr/day operation are: ', ut.sig(total_upfront_cost_24hr,5), ut.sig(total_annual_cost_24hr,5), ut.sig(total_upfront_cost_24hr+total_annual_cost_24hr,5))
print('Upfront, Annual, and Total costs for 12 hr/day operation are: ', ut.sig(total_upfront_cost_12hr,5), ut.sig(total_annual_cost_12hr,4), ut.sig(total_upfront_cost_12hr+total_annual_cost_12hr,5))
print('Upfront, Annual, and Total costs for 8 hr/day operation are: ', ut.sig(total_upfront_cost_8hr,6), ut.sig(total_annual_cost_8hr,4), ut.sig(total_upfront_cost_8hr+total_annual_cost_8hr,6))

Upfront, Annual, and Total costs for 24 hr/day operation are:  38096 USD 17125 USD 55221 USD
Upfront, Annual, and Total costs for 12 hr/day operation are:  90580 USD 8981 USD 99561 USD
Upfront, Annual, and Total costs for 8 hr/day operation are:  109992 USD 6267 USD 116259 USD


## Conclusion

In conclusion, we would recommend that 11 plants, operated for 8 hours a day, be constructed to service the residents of the Siriqui community.  We see 2 main benefits to the 8-hr a day scenario that will be most beneficial to the people of Siriqui.  First, an 8 hour workday is better than a 12 or 24 hour workday.  In the case of one shift per day, an 8 hour day is more ethical than a 12 hour day, and there would be no need to consider splitting the day into multiple shifts, with multiple different operators.  Second, the 11 plant scenario pushes costs upfront in order to minimize the annual costs.  This is beneficial, because upfront costs may be paid for through a fundraiser or donor, whereas annual costs will fall to the unwealthy inhabitants of the island.  We believe that everything that can be done to minimize costs to inhabitants, should be done. Please see calculations below where savings associated with our chosen scenario are calculated.


| Comparison | Annual Savings  | Years to recover initial Deficit   |    
|:----------:|:---------------:|:----------------------------------:|
|8 vs. 24    |     10,900      |           6.6                      | 
|8 vs. 12    |    2,700        |           7.2                      | 
|12 vs. 24   |    8,100        |           6.4                      |


In [10]:
#Compare 12 hour to 24 hour
annual_savings_12_24= total_annual_cost_24hr - total_annual_cost_12hr
initial_deficit_12_24 = total_upfront_cost_12hr - total_upfront_cost_24hr
yrs_recover_12_24 = initial_deficit_12_24/annual_savings_12_24
print('In one year, the 12hr plant saves ' ,annual_savings_12_24, 'compared to the 24hr plant.')
print('Therefore, after ',yrs_recover_12_24, 'years of operation, the initial deficit of the 12 hr plant will be recovered.')

#Compare 8 hour to 24 hour
annual_savings_8_24= total_annual_cost_24hr - total_annual_cost_8hr
initial_deficit_8_24 = total_upfront_cost_8hr - total_upfront_cost_24hr
yrs_recover_8_24 = initial_deficit_8_24/annual_savings_8_24
print('In one year, the 8hr plant saves ' ,ut.sig(annual_savings_8_24,5), 'compared to the 24hr plant.')
print('Therefore, after ',yrs_recover_8_24, 'years of operation, the initial deficit of the 8 hr plant will be recovered.')

#Compare 8 hour to 12 hour
annual_savings_8_12= total_annual_cost_12hr - total_annual_cost_8hr
initial_deficit_8_12 = total_upfront_cost_8hr - total_upfront_cost_12hr
yrs_recover_8_12 = initial_deficit_8_12/annual_savings_8_12
print('In one year, the 8hr plant saves ' ,annual_savings_8_12, 'compared to the 12 hour plant')
print('Therefore, after ',yrs_recover_8_12, 'years of operation, the initial deficit of the 8 hr plant will be recovered')


In one year, the 12hr plant saves  8144 dollar compared to the 24hr plant.
Therefore, after  6.445 dimensionless years of operation, the initial deficit of the 12 hr plant will be recovered.
In one year, the 8hr plant saves  10858 USD compared to the 24hr plant.
Therefore, after  6.621 dimensionless years of operation, the initial deficit of the 8 hr plant will be recovered.
In one year, the 8hr plant saves  2715 dollar compared to the 12 hour plant
Therefore, after  7.151 dimensionless years of operation, the initial deficit of the 8 hr plant will be recovered


<font color="red">
## Disclaimer!
</font>
It should be noted, that near the completion of our project, we received information deeming the lake water unusable. According to David Buck at CH2M Hill, the lake water is too salty, and would need to be treated using an expensive process like reverse-osmosis.  He indicated that future plans will aim to use well water.  The Siriqui well at the moment is non-functional.  When functional, it is not known whether it has the capability of servicing the entire community during the dry season.  The inhabitants currently use spring water which is connected to their distribution system during the rainy season (3 months) and import bottled water for the rest of the year.

Below is a photograph provided by David Buck of the functional distribution system in the Siriqui community.  The well is marked by a light blue circle inside a white box.  There is a pipe (lavender line) connecting the well to the storage tank in Siriqui (white box with blue outline).  An Aguaclara plant could be constructed between the well and the storage tank to treat well water. 

![Image](https://image.ibb.co/c08bvG/unnamed_1.png)

## References

[1] Project Summary and Execution Plan: Siriqui Water Supply Project, Engineers in Action (in folder)

[2] Report of Travel to Siriqui Island for Water Samples Taking (in folder)

[3] Lista de Materiales para la plantita de 1 Lps_Final_08_17 (Excel Spreadsheet, in folder)

[4] Cruse, Andy. “Submersible Pumps vs Surface Mounted Self-Priming Pumps.” Know-How Makes the Difference, ERICKS, 15 May 2015, 09:30, knowhow.eriks.co.uk/submersible-pumps-vs-self-priming-pumps.

[5] David Buck, personal communication

[6] Engineers in Action Bolivia Water Quality Summary (Excel Spreadsheet, in folder)

[7] Professor Monroe Weber-Shirk, Cornell University, personal communication

[8] Juan Guzman, AguaClara consultant, personal communication